### PR TITLE
Bees are now friendly :)

### PR DESCRIPTION
**Pull request by SomeguyManperson**
# General Documentation

### Intent of your Pull Request
bees will no longer attack people on sight, instead they are mostly docile and will mind their own  business unless you try to
A: hit a bee
B: touch the bee hive without proper protection
then they get PISSED for 30 seconds and will try to murder you and everything you love and care about
unless you're wearing  beeproof clothing in which case they will buzz pathetically at you i guess
ALSO bee grenade bees and homeless bees will still act normally i.e. MAD AS HELL constantly so don't worry about buzzkill grenades

### Why is this change good for the game?
makes bees viable and improVES REALISM AND RP
# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
This will be the basis of the Wiki entry for your PR, and more information / detail is better for Wiki editors to integrate.
bee protection is only required for people actually in contact with bees and apiaries, opening the botany window for 0.002 seconds while there are bees will no longer cause 20 of them to try to kill some poor sod in the hallway
### What general grouping does this PR fall under? 
bees
### Are there any aspects of the PR that you would like us not to mention on the Wiki?
bees
### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
bees

# Changelog

:cl:  
tweak: bees are no longer immediately hostile, but hitting bees or trying to steal honey without a bee suit will make them angry
/:cl:

### CODE REVIEW

**Comment by Xoxeyos** [comment]
You should change the bees in the emagged holodeck menu to hostile aswell.

**Comment by Hopekz** [comment]
What about the bees that come out of people?

That whole viro symptom is balanced around the bee attacking the user

**Comment by SomeguyManperson** [comment]
> ALSO bee grenade bees and homeless bees will still act normally i.e. MAD AS HELL constantly so don't worry about buzzkill grenades

**Review by JamieD1 — APPROVED** [review]
